src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Figure implements Drawable, AreaCalculator {
+    private String color;
+    private String name;

```

Comment:
```suggestion
```
the name is always the same for each class that extends Figure class, no need to store it here